src/naive/datetime/tests.rs

```diff
@@ -1,9 +1,37 @@
 use super::NaiveDateTime;
 use crate::oldtime::Duration;
 use crate::NaiveDate;
-use crate::{Datelike, FixedOffset, Utc};
+use crate::{Datelike, FixedOffset, Timelike, Utc};
 use std::i64;
 
+#[test]
+fn test_datetime_from_timestamp_millis() {
+    let timestamp_millis: i64 = 1662921288000; //Sunday, September 11, 2022 6:34:48 PM
```

Comment:
So neither of the test cases you added have non-0 subsecond millis. Maybe add some cases for that so we have actual coverage?

Reply:
good call, I've added in 0 as well as MIN, MAX and some other interesting cases (these do a good test of `from_timestamp_opt` as well